app/assets/stylesheets/content/ui_select.sass

```diff
@@ -1,6 +1,7 @@
 // keep this rule if you want ui-select drag-n-drop to work properly
-.ui-select-match .ui-select-match-item *
-  pointer-events: none
+.ui-select-match .ui-select-match-item
+  span, span *
+    pointer-events: none
```

Comment:
@NobodysNightmare this looks _smelly_ but I'm afraid I'm not familiar with the [`pointer-events` property](https://developer.mozilla.org/en-US/docs/Web/CSS/pointer-events). WDYT @0xF013 @floriank?